new-site/docs-site/content/0.19.0/guide/typesense-dynamodb.md

```diff
@@ -0,0 +1,297 @@
+                // Existing values
+            },
+            "SequenceNumber": "222",
+            "Keys": {
+                // your partion key and sort key
+            },
+            "SizeBytes": 59,
+            "NewImage": {
+                // New Values
+            },
+            "awsRegion": "us-east-2",
+            "eventName": "MODIFY", // this can 'INSERT', 'MODIFY' and 'DELETE'
+            "eventSourceARN": "<AWS-ARN>",
+            "eventSource": "aws:dynamodb"
+        },
+    ]
+}
+```
+
+A sample to code to to process above `event` response
+
+```python
+processed = 0
+for record in event['Records']:
+    ddb_record = record['dynamodb']
+    if record['eventName'] == 'REMOVE':
+        res = client.collections['<collection-name>'].documents[str(ddb_record['OldImage']['id']['N'])].delete()
+    else:
+        upload = ddb_record['NewImage'] # format your document here and the use upsert function to index it.
+        res = client.collections['IPL-Data'].upsert(upload)
+        print(res)
+    processed = processed + 1
+
+print('Successfully processed {} records'.format(processed))
+```
+
+## Deploying a Lambda function (only for Command Line)
+
+* Create a file named `typesense.py ` and add your code into it.
+* Create a zip file to contain `typesense.py`. If you have zip command-line utility
+  ```bash
+  zip typesense-indexing.zip typesense.py
+  ```
+* Get the ARN for the the execution role you created
+  ```bash
+  aws iam get-role --role-name TypesenseLambdaRole
+  ```
+  In output, look for ARN,
+  ```json
+  ...
+  "Arn": "arn:aws:iam::region:role/service-role/WooferLambdaRole"
+  ...
+  ```
+* Now, create the Lambda function
+  ```bash
+  aws lambda create-function \
+    --region us-east-2 \
+    --function-name typesense-indexing \
+    --zip-file fileb://typesense-indexing.zip \
+    --role `roleARN` \
+    --handler typesense-indexing.handler \
+    --timeout 5 \
+    --runtime python3.7
+  ```
+* We need to ARN for DynamoDB to enable trigger for the database
+  ```bash
+  aws dynamodb describe-table --table-name `table-name`
+  ```
+  Note, the ARN for the stream
+  ```json
+  ...
+    "LatestStreamArn": "arn:aws:dynamodb:`region`:`accountID`:table/`table-name`/stream/`timestamp`"
+  ...
+  ```
+* Now, add this ARN to Lambda
+  ```bash
+  aws lambda create-event-source-mapping \
+    --region us-east-1 \
+    --function-name typesense-indexing \
+    --event-source `streamARN ` \
+    --batch-size 1 \
+    --starting-position TRIM_HORIZON
+  ```
+* Make sure to test the function with a set of sample events before enabling the trigger.
+
+That's a wrap! Now your DynamoDB database will be automatically indexed in your Typesense cluster.
+
+## References
+- [Sample Code](https://github.com/HarisaranG/dynamodb-typesense-indexing)
+- [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.Lambda.Tutorial.html)
+- [Typesense API](https://typesense.org/docs/0.19.0/api/)
```

Comment:
Instead of hard-coding the version number, could you use `{{ $page.typesenseVersion }}`